recipe/meta.yaml

```diff
@@ -93,15 +96,25 @@ requirements:
     - setuptools
     - cython >=0.29.22,<3
     - fastrlock >=0.5
-    - cudnn
-    - nccl  # [linux]
-    - cutensor  # [win or cdt_name == "cos7"]
-    {% if cuda_major == 11 %}
-    - cusparselt  # [linux64]
-    {% endif %}
-    {% if cuda_major_minor >= (10, 1) %}
-    - libcugraph  # [linux64]
-    {% endif %}
+    #- cudnn
+    #- nccl  # [linux]
+    #- cutensor  # [win or cdt_name == "cos7"]
+    #{% if cuda_major == 11 %}
+    #- cusparselt  # [linux64]
+    #{% endif %}
+    #{% if cuda_major_minor >= (10, 1) %}
+    #- libcugraph  # [linux64]
+    #{% endif %}
+    - cuda-nvcc  # for crt/host_defines.h
+    - cuda-cudart
+    - cuda-nvrtc
+    - cuda-nvprof  # for cuda_profiler_api.h
```

Comment:
If we have the `cuda-profiler-api` package, does that solve this need? Or are there other things that `cuda-nvprof` is still needed for?

Reply:
I think CuPy doesn't need the legacy `nvprof` (cmdline tool), just the profiler header.